### ISSUE TITLE
--stream output to console is being cropped

### ISSUE THREAD

**Issue by HeavenVolkoff** [syscl/gkernlog#5]
For some reason on `--stream` mode, Console doesn't read the full output and crop some of the last messages in half (they do show up later, when new messages arrive).
Maybe redirecting the output through the file system isn't a good idea to do real-time stream to Console... I will try to come up with a better solution...

**Comment by MuntashirAkon** [comment]
Yeah, just like `log stream >> /tmp/system.log`.

**Comment by HeavenVolkoff** [comment]
Yeah, I tried using pipes to ditch the file system, but it yield the same result... I think `log` has some buffering bug when in stream mode, but I didn't look further into that yet.

**Comment by syscl** [comment]
Hi @HeavenVolkoff , I just saw you have done some really great polish in maclog yourself, wanna post a post request to this project?

Thank you!

syscl
